dash/development/build_process.py

```diff
@@ -33,7 +33,7 @@ def _parse_package(self, path):
             package = json.load(fp)
             self.version = package["version"]
             self.name = package["name"]
-            self.build_folder = self._concat(self.main, self.name.replace("-", "_"))
+            self.build_folder = self._concat(os.getcwd(), os.pardir, "deps")
```

Comment:
Can we avoid involving the CWD here, and do something relative to `self.main`? Then we should be able to avoid `cd`ing into a certain directory before calling `renderer build`, and have it work from any directory.

Reply:
One problem I'm having with using `self.main` to set a relative path is that `build_process` is called from and points to the `site-packages/dash/dash-renderer` directory in the Python environment, rather than the development directory. So as a workaround I've used `os.getcwd()` so the build artifacts are properly updated into the working directory that the package is being developed in. I'm not entirely sure if that was different prior to `dash-renderer` being merged partially with `dash`. 

Could we maybe change that by calling `build_process` in the pwd rather than with a `console_scripts` entrypoint?

Reply:
Do you have dash installed with `-e`? ie `pip install -e .`? That use the dev directory as the installed version.

Reply:
:man_facepalming: Ah, that's what it was, I think somewhere between commits I reinstalled it as a tarfile. I'll have that updated shortly.